### ISSUE TITLE
Trouble importing some character sheets into an existing actor for them

### ISSUE THREAD

**Issue by mdonovan5** [crnormand/gurps#1529]
<img width="289" alt="Screen Shot 2022-08-28 at 18 50 25" src="https://user-images.githubusercontent.com/346141/187098818-de9fb21c-e589-49c9-af00-50a4cdcb0c63.png">
[Archive.zip](https://github.com/crnormand/gurps/files/9440656/Archive.zip)
<img width="823" alt="Screen Shot 2022-08-28 at 18 48 50" src="https://user-images.githubusercontent.com/346141/187099075-c0d61c1f-38cd-4c61-9270-c053da99bd9c.png">

When I import in a fresh empty new actor, it works. But next import fails. After that, I cannot open that actor's sheet until I relaunch. Archive has the .gcs files.

**Comment by crnormand** [comment]
I've put in some protection code to allow the actor to get past this situation (where it couldn't recognize the maneuver stored in the actor).   If another error pops up, let's create a new issue.
